### ISSUE TITLE
API acesss question

### ISSUE THREAD

**Issue by bveber** [davetron33/pyxfl#1]
I can't find any information about how to get an API access token or any site that documents the API endpoints. Can you point me in the right direction?

**Comment by davetron33** [comment]
There isn't anything "official" from the XFL, this was all discovered by inspecting the response headers on the stats page using browser tools, along with the endpoints for player and gameday stats.

I was originally attempting to 'scrape' stats into a Google sheets document by using the =importhtml() function. It wasn't until I noticed the page was being rendered with javascript that I decided to look further and inspect the browser responses to see where the data came from and found the page was making requests to an API.

![image](https://user-images.githubusercontent.com/72425600/221435704-9baa6e12-7643-4f88-a0a4-0b9b55b05249.png)

Use this at your own risk, I'm merely assuming since this is publicly facing there shouldn't be any problems querying data programmatically, and I haven't come across any TOS on the XFL's side. So until someone comes along and says "hey, cut it out", here we are.

I'll have more commits to this later this week.
